examples/ecs/component_change_detection.rs

```diff
@@ -15,14 +18,23 @@ fn main() {
 #[derive(Debug)]
 struct MyComponent(f64);
 
-fn setup(mut commands: Commands) {
+fn setup(mut commands: Commands, mut entropy: ResMut<Entropy>) {
+    let seed = entropy.get();
+    let rng = SmallRng::from_seed(seed);
+    commands.insert_resource(rng);
```

Comment:
Inserting resource with commands is not a pattern I want to encourage further. The application delay is the source of frequent end user bugs, and this would work just fine using another approach.

Reply:
Makes sense. I'm new to bevy, can you point me to the supported way?

Reply:
Init_resource and insert_resource on AppBuilder are the way to go :)